week-7/learning-outcomes.md

```diff
@@ -0,0 +1,47 @@
+# Week 7 Learning Outcomes
+
+## REST APIs
+
+### HTTP & REST
+
+- [ ] **We can build an API that returns JSON**
+- [ ] We can build an API that responds to all HTTP methods
+- [ ] We set relevant HTTP headers on our API responses
+- [ ] We can explain what makes a service RESTful
+
+### Server design
+
+- [ ] **We can structure our server code effectively**
+- [ ] We can structure our code to match REST methods
+
+### Express
+
+- [ ] **We can use Express to build an HTTP server**
+- [ ] We can serve our static assets using Express
+- [ ] We can use Express to handle requests for different routes
+- [ ] We can explain what Express middleware is
```

Comment:
How would we review this learning outcome?

Reply:
All of the "we can explain/describe" ones are tough to review. I see those as targeted more at the student themselves (so they know what do and don't understand), but maybe we can find a way.

Reply:
I think this is covered by the more specific LOs right below: if you can use `body-parser` and write your own error-handling middleware then I think you've demonstrated that you understand middleware